src/java.desktop/unix/native/libsplashscreen/splashscreen_sys.c

```diff
@@ -418,8 +391,6 @@ SplashInitPlatform(Splash * splash) {
     pthread_mutex_init(&splash->lock, NULL);
 
     // We should not ignore any errors.
```

Comment:
The 2nd commented out call looks like it might have been doing something better than the default handler,
but not by so much as to matter. 
The comment seems a bit inappropriate now and I'm puzzled why we need to reinstall the default handler
(ie the call taking NULL), since we haven't installed a non-default handler anyway.
I'd be inclined to remove the comment and the call.

Reply:
Thanks. Makes sense, removed.